rest-api/src/main/resources/scheduling/oozie/workflow_template.xml

```diff
@@ -62,7 +62,7 @@
                 --num-executors $confNumExecutors
                 --executor-memory $confExecutorMemory
                 --driver-cores $driverCores
-                --conf $sparkConfQuotesspark.driver.extraJavaOptions=-Dmenas.rest.uri='$menasRestURI' -Dspline.producer.url='$lineageWriteApiUrl' -Dlog4j.configuration='spark-log4j.properties' -Dconformance.mappingtable.pattern='$mappingTablePattern'$sparkConfQuotes $splineModeConf
+                --conf $sparkConfQuotesspark.driver.extraJavaOptions=-Denceladus.rest.uri='$menasRestURI' -Dspline.producer.url='$lineageWriteApiUrl' -Dlog4j.configuration='spark-log4j.properties' -Dconformance.mappingtable.pattern='$mappingTablePattern'$sparkConfQuotes $splineModeConf
```

Comment:
dtto: Feel free to leave this one for the subsequent PR that solves the field names if you feel it rather belongs there.

```suggestion
                --conf $sparkConfQuotesspark.driver.extraJavaOptions=-Denceladus.rest.uri='$enceladusRestURI' -Dspline.producer.url='$lineageWriteApiUrl' -Dlog4j.configuration='spark-log4j.properties' -Dconformance.mappingtable.pattern='$mappingTablePattern'$sparkConfQuotes $splineModeConf
```